R/bs-theme.R

```diff
@@ -231,14 +229,22 @@ is_bs_theme <- function(x) {
 # Start an empty bundle with special classes that
 # theme_version() & theme_bootswatch() search for
 bs_theme_init <- function(version, subclass = NULL) {
```

Comment:
Can get rid of subclass?

Reply:
Done in b4eff4fedf4c2cd389be38b9c4fcf3d341a86616